packages/core/src/sequelize-typescript.ts

```diff
@@ -11,7 +11,7 @@ import {
   map,
   splitObject,
 } from '@sequelize/utils';
-import chalk from 'chalk';
+import ansis from 'ansis';
```

Comment:
```suggestion
import { red, cyan } from 'ansis';
```
And related changes for this

Reply:
Hello @WikiRik,

thanks for your suggestion. I have updated the PR using color names.